### DISCUSSION TITLE
key related question for a growing company

### DISCUSSION THREAD

**Discussion by therealello33** [iterate-ch/cyberduck#14664] [Q&A]
hi! 
as of now we are providing our employees with a single mountain duck license, therefore we are purchasing a multiple pack of license.

Do i need, if i run out of licenses, to buy a bigger pack first (ex. 50) in order to have some floating keys or is it possibile to purchase some spare ones later on?

Many thanks

**Comment by dkocher** [comment]
Please forward licensing related questions to [support@mountainduck.io](mailto:support@mountainduck.io).
